src/core/shuffler.js

```diff
@@ -0,0 +1,28 @@
+import { deepCopyGrid, listCoordsMovableTiles, move } from './game';
+
+export const SHUFFLE_DURATION = 1000;
```

Comment:
Why not use parameters  default value inside shuffle and let the caller decide how long he wants to shuffle for ?